src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+
+=== Subscriptions
+
+==== Principle
+
+In order to be notified of events a user must have an active <<_glossary_notification_subscription,*Subscription*>> with an associated active notification method.
+It is possible to be notified via the <<_glossary_notification_portal,*Notification Portal*>>, email
+or <<_glossary_notification_third_party,*Third party*>> which is the execution of a script provided by the user.
+
+All these notification methods can be used simultaneously in the same <<_glossary_notification_subscription,*Subscription*>>.
+
+==== Subscription types
+
+There are three types of <<_glossary_notification_subscription,*Subscriptions*>>:
+
+- Scheduler subscription: To be notified when the <<_glossary_proactive_scheduler ,*Scheduler*>> changes its state
+- Job planner subscription: Currently this subscription can be activated with only one event type "All events"
+that corresponds to different issues the <<_glossary_job_planner,*Job Planner*>> may encounter.
+Check out the <<_job_planner_notifications,Job Planner Notifications>> to look at the full list of events.
+- Job subscriptions: To be notified of events that may occur during <<_glossary_workflow,*Workflow*>> executions.
+
+By default each user has a Scheduler and a Job Planner subscription. To start receiving notifications it is ony required
+to choose notification methods, event types and activate the <<_glossary_notification_subscription,*Subscription*>>.
+
+On the other hand Job subscriptions have to be created in order to receive notifications.
+To create a job subscription, go to the <<_glossary_notification_portal,*Notification Portal*>>,
+job subscriptions tab and click on the _+_ button
+
+image::job-subscription-creation-button.png[align="center"]
+
+Then fill and select the mandatory fields and click on _Save_
+
+image::job-subscription-creation.png[align="center"]
+
+At least one of the fields of <<_glossary_workflow,*Workflow*>> data must be provided to create the subscription, that means it
+is not possible to create a job subscription that will listen for all jobs submitted to the Scheduler.
+
+Two scenarios are possible:
+
+- If is provided only a bucket name, then the user will be notified for all <<_glossary_workflow,*Workflows*>> of the bucket
+- Otherwise there has to be an exact match between the bucket name, project name and workflow name of the subscription
+and the bucket name, project name, workflow name of the workflow you wish to be notified of.
+
+The process is the same for Scheduler and Job_planner subscriptions without the
+need to provide <<_glossary_workflow,*Workflow*>> data. Select notification methods,
+events and create the <<_glossary_notification_subscription,*Subscription*>>.
+
+To modify a subscription simply click on it and do your adjustments.
+
+==== Notification methods
+
+The user must activate at least one notification method to be able to activate a subscription. It defines how the user is notified .
```

Comment:
this section is a bit short, you need to mention, for example that the email notification method must be configured on the server in order to allow email notification (with a link to the appropriate configuration explanation in the admin guide).
Same for 3rd party.